core/src/main/java/de/danoeh/antennapod/core/feed/IntraFeedSortOrder.java

```diff
@@ -0,0 +1,31 @@
+package de.danoeh.antennapod.core.feed;
+
+/**
+ * Provides sort orders to sort a list of episodes within a feed.
+ */
+public enum IntraFeedSortOrder {
+    DATE_OLD_NEW(1),
+    DATE_NEW_OLD(2),
+    EPISODE_TITLE_A_Z(3),
+    EPISODE_TITLE_Z_A(4),
+    DURATION_SHORT_LONG(5),
+    DURATION_LONG_SHORT(6);
```

Comment:
The queue stores the sort order, too. Maybe it's better to re-use the same enum. That will also make it easier to share the dialog between queue, batch editor and feed item list (in a future PR).

Reply:
I went back and forth on reusing the existing `SortOrder` enum, and went with a separate one for a few reasons:
1. Sort order for queues have a few that aren't applicable to podcast screen (smart shuffle, etc.). 
2. The value of the sort order is going to be persisted in the database (and hence the explicit code here), so it really belongs to the domain object. Reusing the existing SortOrder will make make the sortorder in Feed possibly carry iinvalid option (e.g,, smart shuffle)

Semantically, we could have a generic `SortOrder`, and have `IntraFeedSortOrder` and `QueueSortOrder` extend from the base (to solve problem 2). But we can't really extend enum. So another solution would be replacing enums with 3 classes that apprxomiate the enums
- a new generic `SortOrder` (e.g., to resue SortDialog), with 2 subclasses
- a  `IntraFeedSotrOrder` for this PR
- a `QueueSortOrder`, replacing the existing `util.SortOrder` enum.